patches/server/0273-Verify-player-key-early.patch

```diff
@@ -0,0 +1,96 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch seems like unnecessarily high-conflict diff for too little benefit in a normal server scenario